_posts/2016/2016-12-20-cqrs-and-elasticsearch.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "CQRS and Elasticsearch"
```

Comment:
maybe: rdbms-and-elasticsearch-with-cqrs.md?
 
or is that still too long?
@jensfischerhh